potential-exploiters/README.md

```diff
@@ -0,0 +1,26 @@
+# Potential Exploiter
+
+## Description
+
+This agent detects when an address funded from Tornado Cash deploy a contract.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+Describe each of the type of alerts fired by this agent
+
+- NETHFORTA-28
+  - Fired when a contract is deployed from an account funded from Tornado Cash.
+  - Severity is always set to "High".
+  - Type is always set to "Suspicious".
+  - The metadata contains the following fields:
+    - `deployer`: The account funded from Tornado deploying a contract 
+
+## Test Data
+
+The agent behaviour can be verified scanning the following block range
+
+- 13961722 - 13961953. This range include the funded and the deployment related to Defi Dollar Finance incidence `https://twitter.com/defidollar/status/1479718032356241408?s=21`.
```

Comment:
Maybe you can add more details, for example the address of the contract deployer

Reply:
I'll add the transaction hashes when the account is funded and when the contract is created 👍